### DISCUSSION TITLE
Servers and cloud

### DISCUSSION THREAD

**Discussion by iliyofksy** [WeAreMahsaAmini/Core#183] [Introductions]
Hello I am a cloud engineer I have access to many servers I have talked with the owners for give permission to create vpn
We should some how give these vpn's to people.
Also I am outside of Iran and I can buy and send starlink devices.

**Comment by norsnap** [comment]
you can help buy spinning up this service https://github.com/WeAreMahsaAmini/Core/discussions/62

**Reply by hiddify-com** [reply]
@iliyofksy 
The starlink devices is not important. People in Iran can never have it for free.
The most important part is starlink subscription fee.

**Reply by iliyofksy** [reply]
@hiddify 
As we talked to those who used Starlink service, each kit will cost approximately 500$ with the subscription fee of 110$.
These stations will have 200mbps-350mbps, we can share the internet in each building I'm pretty sure this can work for 20 people in each building.

**Reply by hiddify-com** [reply]
Exactly, each building can easily pay 500$ of the device. However, for monthly payment of 110$ it is too much for Iranian.
Currently, Iranian ISPs provide around 100\~300 GB traffic per month.
So If starlink can provide a subscription of 1\~3TB/month for free, It will resolve all the problems of Iranian people. Because they can share the device link price.

Please note that Iranians can not pay by credit card.
